extensions/amp-a4a/0.1/friendly-frame-renderer.js

```diff
@@ -28,15 +28,7 @@ import {installUrlReplacementsForEmbed} from '../../../src/service/url-replaceme
 import {setStyle} from '../../../src/style';
 
 /** @typedef {{
```

Comment:
nit: not inline please. Same to other places
```
/**
 * @typedef
```

Reply:
Done.